spec/rails_helper.rb

```diff
@@ -51,4 +51,7 @@
   # The different available types are documented in the features, such as in
   # https://relishapp.com/rspec/rspec-rails/docs
   config.infer_spec_type_from_file_location!
+
+  require 'coveralls'
+  Coveralls.wear!('rails')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.